Bio/Entrez/Parser.py

```diff
@@ -420,7 +422,7 @@ def parse(self, handle):
         self.parser = None
         if self.element is not None:
             # No more XML data, but there is still some unfinished business
-            raise CorruptedXMLError("Premature end of XML stream")
+            raise CorruptedXMLError("Incomplete XML file?")
```

Comment:
You don't know if it's a file.

Reply:
True, but as per the commit comment, my primary purpose here was to remove the changes where we did not have the author's consent to dual license.

I encourage you to add you preferred wording later - or suggest something here?